RadarSDK/Include/RadarTrackingOptions.h

```diff
@@ -154,6 +154,18 @@ typedef NS_ENUM(NSInteger, RadarTrackingOptionsSyncLocations) {
  */
 @property (nonatomic, assign) BOOL beacons;
 
+/**
+ Determines a radius around the center of each geofence under which tracking should ramp up.
+ */
+
+@property (nonatomic, assign) int rampUpRadius;
```

Comment:
thinking a bit about the naming here, I think we may want to talk about this as "nearby tracking" and name these `nearbyTrackingDistance` and `nearbyTrackingInterval`

Reply:
Interesting... open to it, let's discuss in our 1 on 1